packages/frontend/src/pages/InferenceServers.svelte

```diff
@@ -61,7 +61,20 @@ function createNewService() {
               row="{row}"
               bind:selectedItemsNumber="{selectedItemsNumber}" />
           {:else}
-            <div role="status">There is no services running for now.</div>
+            <div role="status">
+              There is no model service. You can <a
+                href="{'javascript:void(0);'}"
+                class="underline"
+                role="button"
+                title="Create a new Model Service"
+                on:click="{createNewService}">create one now</a
+              >.
+            </div>
+            <p>
+              A model service allows testing models using a playground environment and to connect applications as the
```

Comment:
I would not talk first about the Playground as it is _client_, but rather the OpenAPI like and how creating an inference server is exposing an endpoint to let the user connects and experiment with it while providing code snippets.

Reply:
@axel7083 what do you think of the following paragraph?

A model service exposes a configurable endpoint that allows connecting your AI applications
              with the model via a well-known chat API. Once started, you can access the details of the
              model service and inspect code snippets for numerous tools and programming languages to aid
              in developing your applications.

Reply:
Here is a proposition based on the paragraph you provided

````
A model service offers a configurable endpoint via an OpenAI-compatible web server, facilitating seamless integration of AI capabilities into existing applications. Upon initialization, effortlessly access detailed service information and generate code snippets in multiple programming languages to ease application integration.
````

Reply:
Sounds great! Updated.